<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorParametersDialog</class>
 <widget class="QDialog" name="colorParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="brightnessSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>170</number>
   </property>
   <property name="maximum">
    <number>190</number>
   </property>
   <property name="sliderPosition">
    <number>175</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="redSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>116</number>
   </property>
   <property name="maximum">
    <number>129</number>
   </property>
   <property name="sliderPosition">
    <number>120</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="yellowSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>103</number>
   </property>
   <property name="maximum">
    <number>116</number>
   </property>
   <property name="value">
    <number>110</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>亮度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>data1(红)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>66</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>data4(蓝黄)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>170</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>190</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>116</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>129</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>103</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>116</string>
   </property>
  </widget>
  <widget class="QLabel" name="brightnessLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>175</string>
   </property>
  </widget>
  <widget class="QLabel" name="redLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>120</string>
   </property>
  </widget>
  <widget class="QLabel" name="yellowLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>110</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>72</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>色彩优化参数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="toleranceLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>32</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="featherLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>tolerance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>feather</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>colorParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>colorParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
